plugins/modules/cloud/misc/proxmox_kvm.py

```diff
@@ -185,6 +208,19 @@
       - C(size) is the size of the disk in GB.
       - C(format) is the drive's backing file's data format. C(qcow2|raw|subvol).
     type: dict
+  ipconfig:
+    description:
+      - 'cloud-init: Set the IP configuration.'
+      - A hash/dictionary of network ip configurations. C(ipconfig='{"key":"value", "key":"value"}').
+      - Keys allowed are - C(ipconfig[n]) where 0 ≤ n ≤ network interfaces.
+      - Values allowed are -  C("[gw=<GatewayIPv4>] [,gw6=<GatewayIPv6>] [,ip=<IPv4Format/CIDR>] [,ip6=<IPv6Format/CIDR>]").
+      - 'cloud-init: Specify IP addresses and gateways for the corresponding interface.'
+      - IP addresses use CIDR notation, gateways are optional but need an IP of the same type specified.
```

Comment:
I'm not sure about this change, but "IP of the same type" could be confusing.
```suggestion
      - IP addresses use CIDR notation, gateways are optional but they should be in the same subnet of specified IP address.
```